inst/templates/test_memberships.R

```diff
@@ -13,7 +13,7 @@ test_that("missing observations are reported correctly", {
 # Uniformity tests (agreements have a source ID, a string title, a signature and
 # entry into force date)
 test_that("datasets have the required variables", {
-  expect_col_exists({{{dab}}}[["{{{dat}}}"]], vars(Country))
+  expect_col_exists({{{dab}}}[["{{{dat}}}"]], vars(CountryID))
```

Comment:
Should it not be a column that either starts with or end or contain the word "ID"? CountryID sounds very specific...